sfs/array.py

```diff
@@ -43,7 +43,7 @@ def take(self, indices):
 
 
 def linear(N, spacing, center=[0, 0, 0], orientation=[1, 0, 0]):
-    """Linear secondary source distribution.
+    """Get linear, equidistantly sampled secondary source distribution.
```

Comment:
This is better than before, but probably we should use "Return" instead of "Get"?

See https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
all changed to return